test/dlc_tests/ec2/test_transformerengine.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import pytest
+
+import test.test_utils.ec2 as ec2_utils
+from test.test_utils import CONTAINER_TESTS_PREFIX, is_pr_context, is_efa_dedicated
+from test.test_utils.ec2 import get_efa_ec2_instance_type, filter_efa_instance_type
+
+PT_TE_TESTS_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "transformerengine", "testPTTransformerEngine"
+)
+
+
+EC2_EFA_GPU_INSTANCE_TYPE_AND_REGION = get_efa_ec2_instance_type(
+    default="p4d.24xlarge",
```

Comment:
did you test this change on P5? If not it should be tested

Reply:
Yes